web/pandas/versions.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "1.5 (dev)",
```

Comment:
I know Django just uses the label `dev` in their navbar. Might be more flexible to label this as dev since the next version could be to be determined?

https://docs.djangoproject.com/en/4.0/

Reply:
I think we can take whatever we like most. Both options would be fine for me. 
(we need to update this file after a release anyway, so having the version here only means editing one more line)

Reply:
I'd be more partial to just `dev`. Plus it would be one less line to remember to update :)